test/integration/testdata/golden.json

```diff
@@ -402,7 +402,7 @@
     "x-kubernetes-unions": [
      {
       "discriminator": "unionType",
-      "fields-discriminated": {
+      "fields": {
```

Comment:
I think that just plain "fields" is going to be confusing, which is first, the field or its discriminator? How do I even know the second value is a discriminator?

How about "fields-to-discriminator-value"? It's really wordy :/ but I think just "fields" doesn't help people enough.

field-names? fields-to-names?

Reply:
Out of all these options, I like `fields-discriminated` most :-)

Reply:
Let's use `fields-to-discriminatedBy`?

Reply:
Renamed, PTAL

Reply:
renamed to `fields-to-discriminateBy`